docs/quickstart.md

```diff
@@ -78,23 +78,36 @@ cd /tmp
 $ curl https://raw.githubusercontent.com/nix-community/disko/master/example/hybrid.nix -o disko-config.nix
 ```
 
-### Step 5: Run disko to partition, format and mount your disks
+### Step 5: Adjust the device in the disk configuration
 
-The following step will partition and format your disk, and mount it to `/mnt`.
+Inside the disko-config.nix the device needs to point to the correct disk name.
 Replace `<disk-name>` with the name of your disk obtained in Step 1.
 
-Please note: This will erase any existing data on your disk.
+```
+nano disko-config.nix
+```
 
 ```
-$ sudo nix run github:nix-community/disko -- --mode disko /tmp/disko-config.nix --arg disks '[ "/dev/<disk-name>" ]'
+...
+      main = {
+        type = "disk";
+        device = "<disk-name>";
+        content = {
+          type = "gpt";
+...
 ```
 
-For example, if the disk name is `nvme0n1`, the command would be:
+### Step 5: Run disko to partition, format and mount your disks
```

Comment:
```suggestion
### Step 6: Run disko to partition, format and mount your disks
```

Reply:
Need to adapt all the other step numbers as well, or just remove the numbering? :)